lambdas/search/index.py

```diff
@@ -78,34 +78,43 @@ def lambda_handler(request):
         _source = []
     elif action == 'images':
         body = {
-            'query': {
-                'terms': {
-                    "ext": IMG_EXTS
-                }
-            }
+            'query': {'terms': {'ext': IMG_EXTS}},
+            'collapse': {
+                'field': 'key',
+                'inner_hits': {
+                    'name': 'latest',
+                    'size': 1,
+                    'sort': [{'last_modified': 'desc'}],
+                    '_source': ['key', 'version_id'],
+                },
+            },
         }
         size = NUM_PREVIEW_IMAGES
         _source = []
-    elif action == 'other':
+    #TODO: deprecate 'other'
```

Comment:
`other` should only be used by the catalog so go ahead and remove that now?

Reply:
removed